src/workloads/execution/locf.yml

```diff
@@ -0,0 +1,295 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of $locf. The benchmark operations test numeric,
+  string, date and randomized data types. There is a test without partitions, with a
+  single partition, and with multiple partitions for each data type.
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: &batchSize 100
+    Document:
+      part1: {^Choose: {from: [1, 2, 3, 4]}}
+      part2: {^Choose: {from: [1, 2, 3, 4]}}
+      numeric: {^Choose: {from: [
+        {^RandomInt: {min: -100, max: 100}},
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        null], weights: [1, 1, 8]}}
+      string: {^Choose: {from: [
+        {^RandomString: {length: 15}},
+        null], weights: [2, 8]}}
+      date: {^Choose: {from: [
+        {^RandomDate: {min: "2021-01-01", max: "2023-01-01"}},
+        null], weights: [2, 8]}}
+      randomType: {^Choose: {from: [
+        {^RandomInt: {min: -100, max: 100}},
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        {^RandomDate: {min: "2021-01-01", max: "2023-01-01"}},
+        {^RandomString: {length: 15}},
+        null], weights: [1, 1, 1, 1, 5]}}
+  - Nop: true
+  - Nop: true
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+
+- Name: Locf
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Repeat: 10
+    Database: *db
```

Comment:
As written this will run each operation sequentially through, and then repeat that 10 times. Is that what you want? Or would you prefer to have the  operations in different phases with a quiesce between them? 

Also, is 10 repetitions sufficient? Would you prefer to run it for a period of time?

Reply:
We'd love to get your expert opinion here as we don't know what's best. We could definitely pivot to having each operation be in its own phase with a quiesce between them, if you think that's better

Reply:
It depends what you are trying to learn about the system. I suspect you are more interested in the performance of the individual operations, as opposed to their interactions. If that is correct I would encourage you to put them into separate phases.

Reply:
This is now done?

Reply:
It looks like it. Nice. Is 10 reptitions giving you everything that you need here? Or would you prefer a duration? You can use both and it will wait for both to be satisfied. 

https://github.com/mongodb/genny/blob/3cb776c6f5a21056c3c779ae8eafd0f47b8b5b94/docs/HitchhikersGuide.md

> If both the Repeat and Duration keywords are specified, then it will run for the longer of the two (e.g. { Duration: 2 seconds, Repeat: 1 } the Duration will win unless the single iteration takes longer than 2 seconds).

Reply:
We think 10 repetition is giving us sufficient info and we do not need a duration - thank you for providing us with this extra info, will be helpful in the future!

Reply:
The working patch build helps. Thanks. I can see that the operations are taking ~20-40ms on average, so running 10 repetitions is still under a second per operation, and the entire genny invocation is taking only 17 seconds. I think we should add a Duration for these. Maybe 30 seconds?  It just gives us a lot more data and chance to smooth out any weird behavior. 30 seconds is a fairly arbitrary number. For things that drive IO and checkpoints we would want each individual test to be some multiple of the WT checkpoint frequency (minute), but that's probably not necessary here.